server/webhook.go

```diff
@@ -167,11 +167,30 @@ func (s *Server) registerWebhookRoutes(g *echo.Group) {
 				}
 
 				// Create schema update issue.
-				issue, err := s.createSchemaUpdateIssue(ctx, repository, mi, vcsPushEvent, commit, added, string(b))
+				var createContext string
+				if repository.Project.TenantMode == api.TenantModeTenant {
+					createContext, err = s.createTenantSchemaUpdateIssue(ctx, repository, mi, vcsPushEvent, commit, added, string(b))
+				} else {
+					createContext, err = s.createSchemaUpdateIssue(ctx, repository, mi, vcsPushEvent, commit, added, string(b))
+				}
 				if err != nil {
 					createIgnoredFileActivity(err)
 					continue
 				}
+				issueCreate := &api.IssueCreate{
+					ProjectID:     repository.ProjectID,
+					Name:          commit.Title,
+					Type:          api.IssueDatabaseSchemaUpdate,
+					Description:   commit.Message,
+					AssigneeID:    api.SystemBotID,
+					CreateContext: createContext,
+				}
+				issue, err := s.createIssue(ctx, issueCreate, api.SystemBotID)
+				if err != nil {
+					createIgnoredFileActivity(fmt.Errorf("Failed to create update schema issue for added repository file %q, error %v", added, err))
+					continue
```

Comment:
So you still prefer to continue instead of just abort here? (because I am thinking if we fail to createIssue, then there exists serious underlying issue and we are unlikely to proceed anyway).